### ISSUE TITLE
telemeter-wasm-1.0.3 results in a 404

### ISSUE THREAD

**Issue by bsilvereagle** [wiris/html-integrations#793]
## Description

*What does the bug consist in?*

`yarn` is unable to complete on a fresh clone due to a 404 for telemeter-wasm-1.0.3

## Environment

*Which version of MathType does this happen in?*

Commit 7fcb20879bef1a3a037881d242c2321cea326ff1

*What is the relevant software and their versions?*

- docker.io/debian:bookworm-slim

## Steps to reproduce

MVP:

```bash
# git clone https://github.com/wiris/html-integrations.git
# cd mathtype/html-integrations
# yarn
```

Dockerfile:

```
FROM docker.io/debian:bookworm-slim

RUN apt-get update && apt-get install -y git npm && mkdir -p /opt/mathtype && cd /opt/mathtype && git clone https://github.com/wiris/html-integrations.git && cd mathtype/html-integrations 
RUN npm install --global yarn && yarn
```

## Expected result

A successful compile so demos can be launched

## Actual result

```
# yarn
yarn install v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/@wiris/telemeter-wasm/-/telemeter-wasm-1.0.3.tgz: Request failed \"404 Not Found\"".
info If you think this is a bug, please open a bug report with the information provided in "/opt/mathtype/html-integrations/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

## Other details

Removing L55 from `packages/devkit/package.json` and L1 from `packages/devkit/src/telemeter.js` to effectively disable telemeter results in a successful build via `yarn`. However, then this happens:

```
# npx nx@latest start html-generic

 >  NX   Running target start for project html-generic and 1 task(s) it depends on

 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

> nx run html-generic:prestart

yarn run v1.22.19
$ yarn && yarn link @wiris/mathtype-generic && yarn link @wiris/mathtype-html-integration-devkit
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
warning "@nrwl/cypress > @nrwl/devkit@14.8.7" has unmet peer dependency "nx@>= 13.10 <= 15".
warning " > @nrwl/eslint-plugin-nx@14.8.7" has unmet peer dependency "@typescript-eslint/parser@^5.29.0".
warning " > clean-webpack-plugin@4.0.0" has unmet peer dependency "webpack@>=4.0.0 <6.0.0".
warning "workspace-aggregator-306466f3-6138-4d0f-b482-d9df03c70ca2 > @wiris/mathtype-ckeditor4 > css-loader@5.2.7" has unmet peer dependency "webpack@^4.27.0 || ^5.0.0".
warning "workspace-aggregator-306466f3-6138-4d0f-b482-d9df03c70ca2 > @wiris/mathtype-ckeditor4 > url-loader@4.1.1" has unmet peer dependency "webpack@^4.0.0 || ^5.0.0".
warning "workspace-aggregator-306466f3-6138-4d0f-b482-d9df03c70ca2 > mathtype-demos-resources > webpack-cli@3.3.12" has incorrect peer dependency "webpack@4.x.x".
warning "workspace-aggregator-306466f3-6138-4d0f-b482-d9df03c70ca2 > demo-react-ckeditor5 > @ckeditor/ckeditor5-react@2.1.0" has incorrect peer dependency "react@^16.0.0".
warning Workspaces can only be enabled in private projects.
warning Workspaces can only be enabled in private projects.
[4/4] Building fresh packages...
success Using linked package for "@wiris/mathtype-generic".
success Using linked package for "@wiris/mathtype-html-integration-devkit".
Done in 19.49s.

> nx run html-generic:start

<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: http://localhost:8007/
<i> [webpack-dev-server] Content not from webpack is served from '/opt/mathtype/html-integrations/demos/html/generic/' directory

>  NX  Web Development Server is listening at http://0.0.0.0:8007/

asset demo.js 794 KiB [emitted] (name: app)
runtime modules 27.3 KiB 13 modules
javascript modules 654 KiB
  modules by path ./packages/ 412 KiB
    modules by path ./packages/devkit/src/*.js 369 KiB 24 modules
    modules by path ./packages/devkit/styles/ 29.2 KiB 12 modules
    modules by path ./packages/generic/ 13.8 KiB 3 modules
  modules by path ./node_modules/ 238 KiB 33 modules
  modules by path ./demos/html/generic/src/ 4.83 KiB
    ./demos/html/generic/src/app.js 1.42 KiB [built] [code generated]
    ./demos/html/generic/src/static/style.css 2.34 KiB [built] [code generated]
    ./node_modules/css-loader/dist/cjs.js!./demos/html/generic/src/static/style.css 1.07 KiB [built] [code generated]
json modules 40.7 KiB
  ./packages/generic/package.json 1.02 KiB [built] [code generated]
  ./packages/devkit/lang/strings.json 39.6 KiB [built] [code generated]

ERROR in ./demos/html/generic/src/app.js 3:0-53
Module not found: Error: Can't resolve '@wiris/mathtype-generic/wirisplugin-generic' in '/opt/mathtype/html-integrations/demos/html/generic/src'
resolve '@wiris/mathtype-generic/wirisplugin-generic' in '/opt/mathtype/html-integrations/demos/html/generic/src'
  Parsed request is a module
  using description file: /opt/mathtype/html-integrations/demos/html/generic/package.json (relative path: ./src)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /opt/mathtype/html-integrations/demos/html/generic/src/node_modules doesn't exist or is not a directory
      looking for modules in /opt/mathtype/html-integrations/demos/html/generic/node_modules
        existing directory /opt/mathtype/html-integrations/demos/html/generic/node_modules/@wiris/mathtype-generic
          using description file: /opt/mathtype/html-integrations/demos/html/generic/node_modules/@wiris/mathtype-generic/package.json (relative path: .)
            using description file: /opt/mathtype/html-integrations/demos/html/generic/node_modules/@wiris/mathtype-generic/package.json (relative path: ./wirisplugin-generic)
              no extension
                Field 'browser' doesn't contain a valid alias configuration
                /opt/mathtype/html-integrations/demos/html/generic/node_modules/@wiris/mathtype-generic/wirisplugin-generic doesn't exist
              .js
                Field 'browser' doesn't contain a valid alias configuration
                /opt/mathtype/html-integrations/demos/html/generic/node_modules/@wiris/mathtype-generic/wirisplugin-generic.js doesn't exist
              .json
                Field 'browser' doesn't contain a valid alias configuration
                /opt/mathtype/html-integrations/demos/html/generic/node_modules/@wiris/mathtype-generic/wirisplugin-generic.json doesn't exist
              .wasm
                Field 'browser' doesn't contain a valid alias configuration
                /opt/mathtype/html-integrations/demos/html/generic/node_modules/@wiris/mathtype-generic/wirisplugin-generic.wasm doesn't exist
              as directory
                /opt/mathtype/html-integrations/demos/html/generic/node_modules/@wiris/mathtype-generic/wirisplugin-generic doesn't exist
      /opt/mathtype/html-integrations/demos/html/node_modules doesn't exist or is not a directory
      /opt/mathtype/html-integrations/demos/node_modules doesn't exist or is not a directory
      looking for modules in /opt/mathtype/html-integrations/node_modules
        existing directory /opt/mathtype/html-integrations/node_modules/@wiris/mathtype-generic
          using description file: /opt/mathtype/html-integrations/node_modules/@wiris/mathtype-generic/package.json (relative path: .)
            using description file: /opt/mathtype/html-integrations/node_modules/@wiris/mathtype-generic/package.json (relative path: ./wirisplugin-generic)
              no extension
                Field 'browser' doesn't contain a valid alias configuration
                /opt/mathtype/html-integrations/node_modules/@wiris/mathtype-generic/wirisplugin-generic doesn't exist
              .js
                Field 'browser' doesn't contain a valid alias configuration
                /opt/mathtype/html-integrations/node_modules/@wiris/mathtype-generic/wirisplugin-generic.js doesn't exist
              .json
                Field 'browser' doesn't contain a valid alias configuration
                /opt/mathtype/html-integrations/node_modules/@wiris/mathtype-generic/wirisplugin-generic.json doesn't exist
              .wasm
                Field 'browser' doesn't contain a valid alias configuration
                /opt/mathtype/html-integrations/node_modules/@wiris/mathtype-generic/wirisplugin-generic.wasm doesn't exist
              as directory
                /opt/mathtype/html-integrations/node_modules/@wiris/mathtype-generic/wirisplugin-generic doesn't exist
      /opt/mathtype/node_modules doesn't exist or is not a directory
      /opt/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory

webpack 5.76.1 compiled with 1 error in 3118 ms
```

**Comment by thevictorwhyte** [comment]
@bsilvereagle Have you been able to find a workaround for this? This error is breaking my production build.

**Comment by carla-at-wiris** [comment]
Hello @bsilvereagle, thank you so much for your report. We are sorry for the inconvenience.

On Friday, we deployed a new version with the fix for your issue, and updated the changes on the master and stable branches.
If you are using the npm packages, the fix is in the version `8.2.6`. If you still face similar issues, please feel free to contact us and downgrade to the version `8.1.1`, which didn't include the changes that cause this issue.
